app/views/admin/basic_projects/show.html.slim

```diff
@@ -0,0 +1,32 @@
+- content_for(:title, @project.display_name)
+
+section.details
+
+  section.block.basic-project-fields class=(@project.valid? ? 'show-view' : 'edit-view')
+    .show-actions
+      - if policy(@project).update?
+        - if @project.valid?
+          a.edit-action.view-element
+            i.fa.fa-pencil.fa-large>
+            = t(".edit")
+          a.show-action.form-element
+            = t("cancel_edit")
+        - else
+          a href=admin_basic_project_path(@person) = t("cancel_edit")
+      - if policy(@project).destroy?
+        = link_to [:admin, @project], method: :delete,
+            data: { \
+              confirm: t('.confirm_deletion'),
+              "confirm-title": t('.delete'),
+              "confirm-cancel": t(:cancel),
+              "confirm-proceed": t(:confirm)\
+          }
+          i.fa.fa-trash.fa-large>
+          = t(".delete")
```

Comment:
This edit and delete link code should really be refactored into a helper method too. I bet almost the exact same code shows up in about 10 places. The only thing different would be the object. We could include this in our form refactor job. Let's pair on this next week.